spec/tapioca/ruby_lsp/gem_rbi_check_spec.rb

```diff
@@ -57,10 +57,11 @@ class RunGemRbiCheckSpec < SpecWithProject
           @project.require_mock_gem(foo)
           @project.bundle_install!
 
-          check = ::RubyLsp::Tapioca::RunGemRbiCheck.new(@project.absolute_path)
-          check.run
+          gems_for_tapioca_command = []
```

Comment:
If we had Mocha we could write this as something like `check.expects(:run).yields(["foo"])`